uc-certificate-fix/ucCertRemediation.groovy

```diff
@@ -95,6 +111,16 @@ _retry_time = 30000;   // how long to wait before checking for an update site to
 _cert_error_str = "CertificateExpiredException: NotAfter: Tue Oct 19 14:31:36 EDT 2021";
 
 // MAIN CODE BODY
+info("Checking for first run inside an OCI container");
+// The Dockerfile will need to create this file. It will prevent this script from running.
+noRunFilePath = "/var/jenkins_home/init.groovy.d/DO_NOT_RUN_UC_REMEDIATION";

```

Comment:
I would store this marker file outside of the init.groovy folder in case jenkins tries to "run" it during the initial install